README.md

```diff
@@ -194,21 +190,22 @@ android {
 </details>
 
 
-### Configuring reports
-If you need to change the name of the XML report file or HTML directory, you may configure the corresponding tasks
+### Configuring aggregated reports
+If you need to change the name of the XML report file or HTML directory, you may configure the corresponding tasks in 
+the module in which the plugin is applied (usually this is the root module).
```

Comment:
I think we need to add here some kind of definition for an aggregated report. E.g. 'aggregated report provides report using combined classpath from all subprojects and available when plugin is applied to the root project' or something like that